File: app/src/main/java/mx/dev/franco/automusictagfixer/room/database/TrackContract.java
package mx.dev.franco.automusictagfixer.room.database;

import android.provider.BaseColumns;

/**
 * Created by franco on 7/03/17.
 */

public final class TrackContract {
    public TrackContract(){}

    public static class TrackData implements BaseColumns{
        public static final String TABLE_NAME = "tracks_table";
        public static final String MEDIASTORE_ID = "media_store_id";
        public static final String TITLE = "title";
        public static final String ARTIST = "artist";
        public static final String ALBUM = "album";
        public static final String DATA = "_data";
        public static final String IS_PROCESSING = "is_processing";
        public static final String IS_SELECTED = "is_selected";
        public static final String STATUS = "status"; //P for "processed", I for "incomplete data", NA for "no data avaliable", N for "no processed"
    }

}

File: app/src/main/java/mx/dev/franco/automusictagfixer/persistence/room/database/TrackContract.kt
package mx.dev.franco.automusictagfixer.persistence.room.database

import android.provider.BaseColumns

/**
 * Created by franco on 7/03/17.
 */
class TrackContract {
    object TrackData : BaseColumns {
        const val TABLE_NAME = "tracks_table"
        const val MEDIASTORE_ID = "media_store_id"
        const val TITLE = "title"
        const val ARTIST = "artist"
        const val ALBUM = "album"
        const val DATA = "_data"
        const val IS_PROCESSING = "is_processing"
        const val IS_SELECTED = "is_selected"
        const val STATUS =
            "status" //P for "processed", I for "incomplete data", NA for "no data avaliable", N for "no processed"
    }
}